core/server/worker/src/main/java/alluxio/worker/block/BlockMasterClient.java

```diff
@@ -137,18 +139,20 @@ public Command call() throws TException {
    * @param totalBytesOnTiers mapping from storage tier alias to total bytes
    * @param usedBytesOnTiers mapping from storage tier alias to used bytes
    * @param currentBlocksOnTiers mapping from storage tier alias to the list of list of blocks
+   * @param configList a list of configurations
    */
   // TODO(yupeng): rename to workerBlockReport or workerInitialize?
   public synchronized void register(final long workerId, final List<String> storageTierAliases,
       final Map<String, Long> totalBytesOnTiers, final Map<String, Long> usedBytesOnTiers,
-      final Map<String, List<Long>> currentBlocksOnTiers) throws IOException {
-    retryRPC(new RpcCallable<Void>() {
-      @Override
-      public Void call() throws TException {
-        mClient.registerWorker(workerId, storageTierAliases, totalBytesOnTiers, usedBytesOnTiers,
-            currentBlocksOnTiers, new RegisterWorkerTOptions());
-        return null;
-      }
+      final Map<String, List<Long>> currentBlocksOnTiers,
+      final List<ConfigProperty> configList) throws IOException {
+    retryRPC(() -> {
+      RegisterWorkerTOptions options = new RegisterWorkerTOptions();
+      options.setConfigList(configList.stream()
+          .map(ConfigProperty :: toThrift).collect(Collectors.toList()));
```

Comment:
google style is no spaces around the `::` in a method reference
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

Reply:
Changed, thanks for the info!